inference-engine/src/transformations/src/transformations/convert_precision.cpp

```diff
@@ -264,6 +266,13 @@ bool fuse_type_to_bucketize(std::shared_ptr<ngraph::Node> & node, ngraph::elemen
 
 bool fuse_type_to_generic_ie(std::shared_ptr<ngraph::Node> & node, ngraph::element::Type to, size_t idx) {
     node->set_output_type(idx, to, node->output(idx).get_partial_shape());
+    auto & rt_info = node->get_rt_info();
+    using attr_type = VariantWrapper<GenericIEConvertPrecision>;
+    if (rt_info.count(attr_type::type_info.name)) {
+        if (auto attr = std::dynamic_pointer_cast<attr_type>(rt_info[attr_type::type_info.name])) {
```

Comment:
Should we remove convert precision after applying from rt information?

Reply:
No, it is just a way how convert precision can be triggered without linking to ie library.